index.html

```diff
@@ -2612,6 +2617,96 @@ <h3 id="applying">
       </section>
       <section>
         <h3>
+          <code title="">category</code> member
+        </h3>
+        <p>
+          The <dfn id="member-category"><code>category</code>
+          member</dfn> describes the expected application category to which the 
+          web application belongs. This is a hint to catalogs or stores of web 
+          applications under which category (or categories) to list this web 
+          application.
+        </p>
+        <p>
+          The <code>category</code> member is only meant as a hint to catalogs 
+          or stores listing web applications and it is expected that these 
+          will make a best effort to find an appropriate category under which 
+          to list the web application.
+        </p>
+        <p>
+          The <dfn>steps for processing the <code>category</code>
+          member</dfn> are given by the following algorithm. The algorithm
+          takes a <var>manifest</var> as an argument. This algorithm returns a
+          string or <code>undefined</code>. The string MAY include a comma 
+          delimited list of categories.
+        </p>
+        <ol>
+          <li>Let <var>value</var> be the result of calling the
+          <a>[[\GetOwnProperty]]</a> internal method of <var>manifest</var>
+          with argument "<code>category</code>".
+          </li>
+          <li>If <a>Type</a>(<var>value</var>) is not "string":
+            <ol>
+              <li>If <a>Type</a>(<var>value</var>) is not "undefined", <a>issue
+              a developer warning</a> that the type is not supported.
+              </li>
+              <li>Return <code>undefined</code>.
+              </li>
+            </ol>
+          </li>
+          <li>Return <var>category</var>.
+          </li>
+        </ol>
+        <p>
+          The category string should be treated as case insensitive. Thus, 
+          <code>sports</code>, <code>Sports</code>, <code>SPORTS</code>, and 
+          <code>SpOrTs</code> are all equivalent.
+        </p>
+        <p>
+          Spaces, underscores, and dashes should be treated as equivalent. 
+          Thus, <code>health & fitness</code>,  
+          <code>health_&_fitness</code>, and <code>health-&-fitness</code> 
```

Comment:
I wonder if we can avoid this & altogether. We already allow to add multiple categories, so one store could put "health" in the "Health and Fitness" group, and you could event set the category to "health, fitness" and the store could use both values to decide whether to add to its "Health and Medicin" or the "Health and Fitness" category.